static/js/vue-cdr-access/src/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
Is this sort of the top level definition of the browse component? Would it make sense to name it accordingly, or is this file going to be used for all vue components in the access app?

Reply:
In theory all components. We could rename it. It's the generated file name